### PR TITLE
[0.27] Fix CI jobs - Update conan packages

### PR DESCRIPTION
**Pull request by piponazo**

### CODE REVIEW

File: .github/workflows/on_push_BasicWinLinMac.yml [hunk 1]
```diff
@@ -31,7 +31,7 @@ jobs:
 
       - name: Install Conan & Common config
         run: |
-          pip.exe install "conan==1.45.0"
+          pip.exe install "conan==1.48.0"
```
Comment (kmilos) [inline .github/workflows/on_push_BasicWinLinMac.yml:34]: Why are only these different, i.e. 1.48.0 vs 1.48.1 for others?

File: conanfile.py [hunk 2]
```diff
@@ -32,12 +32,12 @@ def requirements(self):
                 self.options['gtest'].debug_postfix = ''
 
         if self.options.webready:
-            self.requires('libcurl/7.80.0')
+            self.requires('libcurl/7.79.0')
```
Comment (kmilos) [inline conanfile.py:35]: The newer version has issues?

File: .github/workflows/on_push_BasicWinLinMac.yml (hunk 1 above)
Comment (piponazo) [reply .github/workflows/on_push_BasicWinLinMac.yml:34]: ups this was a mistake from my part. I wanted to upgrade all of them to 1.48.1. Thanks for noticing

File: conanfile.py (hunk 2 above)
Comment (kmilos) [reply conanfile.py:35]: Sure, why not go for the newer/latest one on both then?

**Review by kmilos — APPROVED** [review]
LGTM

**Pull request merged**